src/modules/fetch-dedupe.ts

```diff
@@ -0,0 +1,93 @@
+// This is a cache of in-flight requests. Each request key maps to an
```

Comment:
Is this file a copy-and-paste of pre-existing work?

If so, can we get a link here of the original, decide if the license is acceptable, and the perhaps add an extra comment header here indicating origin right here.

Reply:
So far, I've just found @jmeas ' _own_ work :) https://github.com/jmeas/react-request/blob/8a57c09be6c3c75df0c6c21f7f72587d01380877/src/fetch-dedupe.js -- but let's get a confirmation and make it explicit in either case.

Thanks!

Reply:
It is from another lib of mine, [request-request](https://github.com/jmeas/react-request/blob/master/src/fetch-dedupe.js), which is licensed under MIT. As the author of the code, I'm OK with no extra header comments here muddying up the source if that's alright with FormidableLabs.

I plan to open sauce the `fetch-dedupe` piece as a separate module, ([npm placeholder here](https://www.npmjs.com/package/fetch-dedupe)) but I haven't gotten around to it. It'll also be MIT licensed.

Whenever that happens (probably in the next 1-2 weeks), this could be updated to depend on that npm module if that is preferable.

Reply:
@jmeas -- Thanks for the response and that sounds fine without extra header comments.